test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -674,7 +677,8 @@ TEST_F(PreparedBlockTest, CommitPreparedFailsAfterCommit) {
 
   auto commited = storage->commitPrepared(block);
 
-  ASSERT_FALSE(commited);
+  ASSERT_TRUE(commited);
```

Comment:
I'm confused a bit

Reply:
this is due to the change described in PR head:
> commitPrepared also returns a CommitResult object, but wrapped into an optional. The optional being unset means that the prepared blocks were not enabled (as it previously was). The rest of errors are now represented with Result errors.
So here prepared commits are enabled, therefore optional must be true. But the commit itself is erroneous, so the `CommitResult` contains an error (next line).

Reply:
Not sure if such complication is required.